src/NUnitFramework/testdata/TestFixtureData.cs

```diff
@@ -121,6 +121,7 @@ public void SomeTest(int x)
     [TestFixture]
     public class FixtureWithParameterizedTestAndArgsSupplied
     {
+        [Test]
```

Comment:
Perhaps I didn't state it clearly enough... I was asking you to remove the redundant `[Test]` attributes.... not add more!